### PR TITLE
Portal cleanups and fixes

### PR DESCRIPTION
**Pull request by sjpadgett**
Move all new template menu to action bar dropdown New Documents
Restyle appointments to cards
Fixes to chat
Synchronize SPA panel collapses
Override theme for ckeditor height!

### CODE REVIEW

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
2DV-2

**Comment by sjpadgett** [comment]
Register is mobile friendly now. Yeah me!

image

image

File: portal/portal_payment.php [hunk 1]
```diff
@@ -1285,7 +1285,7 @@ function getAuth() {
         if (!isset($_SESSION['authUserID'])) {
             if (!isset($ccdata["cardHolderName"])) {
                 if ($GLOBALS['payment_gateway'] == 'Sphere') {
-                    echo SpherePayment::renderSphereHtml('patient');
+                    echo SpherePayment::renderSphereHtml();
```
Comment (bradymiller) [inline portal/portal_payment.php:1288]: gotta revert this line back :)

Comment (sjpadgett) [reply portal/portal_payment.php]: Yep, continue to keep an eye on me @bradymiller and thanks...

**Comment by sjpadgett** [comment]
Tested everything on android and IOS iPhones. All panels appear usable and responsive. Fixed LBF not scrolling in documents.
Some more fine tuning is possible and hopefully someone else will pitch in.
Oh, forgot to restyle appointment dialog and openings so, be right back!:)

**Comment by sjpadgett** [comment]
image

image

image

Okay folks, I'm done!

**Comment by sjpadgett** [comment]
I can't seem to finesse lbf stylings in portal for mobile. Needs to be fixed in core lbf stylings or get rid of lbf in portal.
I'll prob take a stab at it before release of v6.1

**Comment by sjpadgett** [comment]
Tested well bringing in

**Pull request merged**